tools/importer/rules/accordion.js

```diff
@@ -0,0 +1,35 @@
+import { addRow } from '../utils.js';
```

Comment:
You can get rid of this `utils.js`. See below.

Reply:
Removed this file because
`const table = WebImporter.DOMUtils.createTable(cells, document);`
with this it is easy to create the table.